cmd/utils/flags.go

```diff
@@ -202,6 +202,11 @@ var (
 		Usage: "Public address for transaction broadcasting and block mining rewards (default = first account)",
 		Value: "0",
 	}
+	BLSbaseFlag = cli.StringFlag{
```

Comment:
I had a quick chat with @asaj regarding this new flag.  A question that we had is why not always use the converted etherbase address (e.g. why give the option to have a separate BLS base)?

Reply:
The thought was the ability to move only the BLS signatures to a hardware wallet.

Reply:
Sounds good!